docs/tutorial/Blocking with Anonlink Entity Service.ipynb

```diff
@@ -0,0 +1,1632 @@
+      "    \"result_token\": \"63b221ac397bf61a8e78c42c299055110ff7ebcf584b26c6\",\n",
+      "    \"update_tokens\": [\n",
+      "        \"9a113e07d4c05b1ee51f121701e9b0d874e454a7658501fc\",\n",
+      "        \"ea668d9ea3273e42114397897e3f31586aa7e5a1f800878f\"\n",
+      "    ]\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "credentials = json.load(open(\"credentials.json\", 'rt'))\n",
+    "print(json.dumps(credentials, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {}
+   },
+   "source": [
+    "### Uploading the CLKs to the entity service\n",
+    "Each party individually uploads its respective CLKs to the entity service. They need to provide the `resource_id`, which identifies the correct results, and an `update_token`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!anonlink upload \\\n",
+    "       --project=\"{credentials['project_id']}\" \\\n",
+    "        --apikey=\"{credentials['update_tokens'][0]}\" \\\n",
+    "        --output \"upload_a.json\" \\\n",
+    "        --server=\"{SERVER}\" \\\n",
+    "        --blocks=\"blocks_a.json\" \\\n",
+    "        \"clks_a.json\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!anonlink upload \\\n",
+    "       --project=\"{credentials['project_id']}\" \\\n",
+    "        --apikey=\"{credentials['update_tokens'][1]}\" \\\n",
+    "        --output \"upload_b.json\" \\\n",
+    "        --server=\"{SERVER}\" \\\n",
+    "        --blocks=\"blocks_b.json\" \\\n",
+    "        \"clks_b.json\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {}
+   },
+   "source": [
+    "Now that the CLK data has been uploaded the analyst can create one or more *runs*. Here we will start by calculating a mapping with a threshold of `0.9`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[31mConnecting to Entity Matching Server: http://0.0.0.0:8851\u001b[0m\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "!anonlink create --verbose  \\\n",
+    "    --server=\"{SERVER}\" \\\n",
+    "    --output \"run_info.json\" \\\n",
+    "    --threshold=0.5 \\\n",
+    "    --project=\"{credentials['project_id']}\" \\\n",
+    "    --apikey=\"{credentials['result_token']}\" \\\n",
+    "    --name=\"Blocked run\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
```

Comment:
The outputs are all empty from this cell. Maybe restart and run all so that outputs of following cells are available?